.github/workflows/building.yaml

```diff
@@ -7,7 +7,7 @@ on:
       - 'v*'
 
 permissions:
-  contents: read
```

Comment:
I think this action to publish the package in HACS (so the ZIP build) may live in its own GitHub action. What do you think if we have something along these lines:
- `workflows/building.yaml` that includes the building step for a regular package (as it is right now), so that who doesn't use HACS, simply uses this package.
- `workflows/hacs.yaml` that is basically your change!

While both are actually "building a package", this one has the extra step of publishing it. I think it may be better to separate it so that you can change the trigger at the `publish` step instead of `push tags`.

Long story short, what if your change creates `hacs.yaml` that can be inspired by what the [HACS official package](https://github.com/hacs/integration/blob/main/.github/workflows/publish.yml) does?

In this way, instead of triggering this action at `v*` tag, you trigger it when we publish a new package: https://github.com/hacs/integration/blob/main/.github/workflows/publish.yml#L3-L6

Reply:
I agree on all the requests but one, so I went forward and added a separate file